<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数独游戏</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/ksudoku_103845.ico</normaloff>icons/ksudoku_103845.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="level_menu">
    <property name="title">
     <string>难度</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuGam">
    <property name="title">
     <string>游戏</string>
    </property>
    <widget class="QMenu" name="grid_size_menu">
     <property name="title">
      <string>游戏类型</string>
     </property>
    </widget>
    <addaction name="actionNew_Game"/>
    <addaction name="actionRestart_Game"/>
    <addaction name="actionHint_one"/>
    <addaction name="actionHint_All"/>
    <addaction name="grid_size_menu"/>
   </widget>
   <addaction name="menuGam"/>
   <addaction name="level_menu"/>
  </widget>
  <action name="actionNew_Game">
   <property name="text">
    <string>新游戏</string>
   </property>
  </action>
  <action name="actionHint_one">
   <property name="text">
    <string>提示</string>
   </property>
  </action>
  <action name="actionHint_All">
   <property name="text">
    <string>显示答案</string>
   </property>
  </action>
  <action name="actionRestart_Game">
   <property name="text">
    <string>重新开始</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
